### DISCUSSION TITLE
ajout raccourcis

### DISCUSSION THREAD

**Discussion by Lagri85** [File-New-Project/EarTrumpet#1550] [Ideas]
bonjour,
Serait il possible d'ajouter des raccourcis clavier sur votre application afin de gérer le son d'une application (par exemple Spotify) pendant que nous sommes en jeu ?
Merci d'avance

**Comment by riverar** [comment]
Bonjour! C'est ce que nous soutenons dans EarTrumpet Actions ! Essayez nos versions de développement (https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller).

(Hello! We support that in EarTrumpet Actions! Try our development builds (https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller).)
